### DISCUSSION TITLE
Unable to obtain permission

### DISCUSSION THREAD

**Discussion by Lance-Azrael** [community/community#13072] [Copilot Conversations] [answered]
I have joined the waitlist for a long time, but I did not see the response. However, I can see GitHub Copilot Plugin in my GitHub Applications page, as the picture below. Does that mean I'm qualified to use it?
![2K}_)4G`V@H44@@97$@K 17](https://user-images.githubusercontent.com/74921991/159158186-f604aea9-4180-414c-bf9f-5c6e3cc727a8.png)
If I am, why IntelliJ IDEA still tells me that I do not have the access to GitHub Copilot, as the picture below? I have installed it correctly and finish the device connection.
![Z9Y)$B1@B)PA7I7{AR%%T19](https://user-images.githubusercontent.com/74921991/159158280-bb134ed5-5d17-4952-807f-519c971ba9b6.png)
![MG 0DP8{ZV3(_7MLUFHAQ(7](https://user-images.githubusercontent.com/74921991/159158282-1c6ae29d-162d-4315-a39d-a3b015d757f5.png)

**Comment by Lance-Azrael** [comment]
Other group members of mine have gained the response email but they got the same result with me when trying to use it on IntelliJ IDEA. It still says that they don't have the access. I wonder if I miss some steps before we can use it successfully.

**Comment by davecheney** [comment, marked as answer]
@Lance-Azrael I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
